cypress/e2e/expenses/Add.cy.js

```diff
@@ -0,0 +1,105 @@
+import '@testing-library/cypress/add-commands';
+
+import {
+  getUKFormattedEuros,
+  getUKFormattedDate,
+  getCurrentMonth,
+} from '../../../src/utils/helpers';
+
+describe('ADD EXPENSE - NOT CURRENT MONTH', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', []).as('getExpenses');
+    cy.visit('/');
+    cy.wait('@getExpenses');
+  });
+
+  beforeEach(() => {
+    cy.intercept('POST', '**/items*', { fixture: 'expense-post' }).as(
+      'postExpense'
+    );
+  });
+
+  it('adds the expense', () => {
+    cy.addExpenseFromFixture('expense-raw');
+    cy.wait('@postExpense');
+  });
+
+  it('selects expense month', () => {
+    cy.fixture('expense-raw').then(expense => {
+      const expenseMonth = getUKFormattedDate(expense.date, {
```

Comment:
not sure if it is cool to use an internal function here. aren't we supposed to test this along with the rest?

Reply:
sorry not sure I got it right. you mean that we should not use our helper functions in the tests, but create something like separate cypress helpers to be used in the tests?

Reply:
well, i am really unsure. it feels wrong to use the internal helpers. what about hardcoding the value here?

Reply:
all values are now hardcoded